features/options/contract/edge-weight-updates-over-factor.feature

```diff
@@ -33,15 +33,25 @@ Feature: osrm-contract command line option: edge-weight-updates-over-factor
 
 
     Scenario: Logging using weigts as durations for non-duration profile
-        Given the profile file "testbot" extended with
+        Given the profile file
```

Comment:
Is this a necessary change with the new API? I would hate to maintain these in addition to the testbot profile, if only a specific entry should change. In my view we should maintain as much as we can within the main profile and only adapt what is absolutely necessary.